legislation_analysis/processing/clean.py

```diff
@@ -173,6 +190,11 @@ def is_valid_word(cls, word: str) -> bool:
 
         word = word.lower()
 
+        # check if word is a name
+        potential_name = cls.NAMES_DATASET.search(word)
+        if potential_name["first_name"] or potential_name["last_name"]:
```

Comment:
I think this should be `if "first_name" in potential_name or "last_name" in potential_name:`

Reply:
naur, potential_name returns a dictionary of {"first_name": [list of potential first names], "last_name": [list of potential last names]} and if the list is empty, then None for those